test.js

```diff
@@ -0,0 +1,3 @@
+require('dotenv').load()
+var test = require('./scripts/musicbrainz-initial-dump/importArtists.js')
+test.myHandler()
```

Comment:
This whole script is only for testing at this point, I'll remove or replace it once I figure out what the best setup is for Lambda functions.